test/controllers/short_problems_controller_test.rb

```diff
@@ -20,7 +20,7 @@ class ShortProblemsControllerTest < ActionController::TestCase
     test_abilities @sp, :create, [nil, :panitia], %i[problem_admin admin]
     post :create, contest_id: @c.id,
                   short_problem: { statement: 'Hello there',
-                                   problem_no: 5,
+                                   problem_no: 1000,
```

Comment:
Don't think this change is needed

Reply:
Please check why changing this to 1000 solves this test

Reply:
Sorry, this is my fault. There is no wrong with 5. Once again, I'm so sorry.